packages/opencensus-exporter-ocagent/test/test-ocagent.ts

```diff
@@ -331,13 +334,16 @@ describe('OpenCensus Agent Exporter', () => {
       rootSpan.setStatus(CanonicalCode.OK);
 
       // Attribute
+      rootSpan.addAttribute('my_first_attribute', 'foo');
+      rootSpan.addAttribute('my_second_attribute', 'foo2');
       rootSpan.addAttribute('my_attribute_string', 'bar2');
       rootSpan.addAttribute('my_attribute_number', 456);
       rootSpan.addAttribute('my_attribute_boolean', false);
```

Comment:
Change the test to add same key again such that it is no longer oldest and hence should not be deleted.

Reply:
Done, PTAL